src/global/interfaces/pearl-chain-properties.ts

```diff
@@ -72,38 +66,338 @@ export interface ScheduledStopPoint {
   stopStatus?: StopStatusEnum;
 }
 
-/** A passenger carrying vehicle journey for one specified operation day */
-export interface ServiceJourney {
-  /** List of ServiceProduct attributes and journey hints */
-  /** Status (realtime changes) to Journey. */
-  serviceAlteration: ServiceAlteration;
-  /** List of stop points */
-  stopPoints: ScheduledStopPoint[];
-  /** List of ServiceProduct attributes and journey hints */
-  notices: Notice[];
+export enum BoardingAlightingAccessibilityEnum {
+  BOARDING_ALIGHTING_BY_CREW = 'BOARDING_ALIGHTING_BY_CREW',
+  BOARDING_ALIGHTING_BY_NOTIFICATION = 'BOARDING_ALIGHTING_BY_NOTIFICATION',
+  BOARDING_ALIGHTING_NOT_POSSIBLE = 'BOARDING_ALIGHTING_NOT_POSSIBLE',
+  BOARDING_ALIGHTING_SELF = 'BOARDING_ALIGHTING_SELF',
+}
+
+export enum PtSituationCauseEnum {
+  CONSTRUCTION_SITE = 'CONSTRUCTION_SITE',
+  DELAY = 'DELAY',
+  DISTURBANCE = 'DISTURBANCE',
+  END_MESSAGE = 'END_MESSAGE',
+  INFORMATION = 'INFORMATION',
+  TRAIN_REPLACEMENT_BY_BUS = 'TRAIN_REPLACEMENT_BY_BUS',
+}
+export enum OccupancyEnum {
+  HIGH = 'HIGH',
+  LOW = 'LOW',
+  MEDIUM = 'MEDIUM',
+  UNKNOWN = 'UNKNOWN',
+}
+
+/** Type of Notice */
+export enum NoticeTypeEnum {
+  ATTRIBUTE = 'ATTRIBUTE',
+  INFO = 'INFO',
 }
 
-export interface Leg {
-  /** duration from previous transfer point to current in minutes */
-  duration?: number;
+export enum TextArgumentEnum {
+  EMAIL = 'EMAIL',
+  PHONE = 'PHONE',
+  URL = 'URL',
 }
 
-export type PtRideLeg = {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  __typename?: 'PTRideLeg';
-  /** arrival time and quay */
-  arrival: ScheduledStopPointDetail;
-  /** departure time and quay */
-  departure: ScheduledStopPointDetail;
-  duration?: number;
-  /** (partial) journey details */
-  serviceJourney: ServiceJourney;
+/** key-value pair. Key is one of: (EMAIL,PHONE,URL) */
+export type TextArgument = {
+  __typename?: 'TextArgument';
+  type?: TextArgumentEnum;
+  values: string[];
+};
+
+/** Mode of public transportation */
+export enum VehicleModeEnum {
+  BUS = 'BUS',
+  CABLEWAY = 'CABLEWAY',
+  CHAIRLIFT = 'CHAIRLIFT',
+  COG_RAILWAY = 'COG_RAILWAY',
+  ELEVATOR = 'ELEVATOR',
+  GONDOLA = 'GONDOLA',
+  METRO = 'METRO',
+  PLANE = 'PLANE',
+  SHIP = 'SHIP',
+  TAXI = 'TAXI',
+  TRAIN = 'TRAIN',
+  TRAMWAY = 'TRAMWAY',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export type GetTripsQuery = {
```

Comment:
this looks... not readable :D It's also generating massive documentation output for timetable-row readme. Did you copy that from the backend api?

Reply:
I assume this is generated from OpenApi or something similar? I would MUCH prefer this be split into separate interfaces.

Reply:
Hey guys, I split the definitions. I hope this works better for us :D

I tested it in our frontend. and it works fine